superdesk.config.js

```diff
@@ -18,6 +18,7 @@ module.exports = function(grunt) {
             validatePointOfInterestForImages: true,
             hideRoutedDesks: false,
         },
+        iframely: {key: '4358cefd09a66b1126c036'},
```

Comment:
should the key be here? I assume it's free, but if we start deploying it by default it won't work eventually

Reply:
Yep, it shouldn't. I've added it for testing and forgot to remove it.